core/domain/rte_component_registry_test.py

```diff
@@ -209,7 +219,9 @@ class RteComponentRegistryUnitTests(test_utils.GenericTestBase):
     def test_get_all_rte_components(self):
         """Test get_all_rte_components method."""
         obtained_components = (
-            rte_component_registry.Registry.get_all_rte_components().keys())
+            list(
```

Comment:
Move list( to the previous line and get rid of the outer parens.

Reply:
Done